griptape/tokenizers/base_tokenizer.py

```diff
@@ -17,6 +29,32 @@ def count_tokens_left(self, text: str | list) -> int:
         else:
             return 0
 
+    def count_output_tokens_left(self, text: str | list) -> int:
+        diff = self.max_output_tokens - self.count_tokens(text)
+
+        if diff > 0:
+            return diff
+        else:
+            return 0
+
     @abstractmethod
     def count_tokens(self, text: str | list[dict]) -> int:
```

Comment:
We should keep this as `count_tokens`, right? Since it can be used in any context.

Reply:
Correct